.github/workflows/check-links.yml

```diff
@@ -32,6 +32,21 @@ jobs:
         args: >
           --accept 429
           --exclude "http://localhost:1313/"
+          --exclude "^https://data.earthquake.cn/datashare/report.shtml?PAGEID=earthquake_lsdz$"
```

Comment:
Removing `^` and `$` should also work. It's simpler.
```suggestion
          --exclude "https://data.earthquake.cn/datashare/report.shtml?PAGEID=earthquake_lsdz"
```

Reply:
Not really. I did not use them at the beginning.

Reply:
OK, I see, but adding `^` and `$` also doesn't work, right?

Reply:
Yep. Confused~